test/tests/autotranslation.js

```diff
@@ -18,11 +18,16 @@ var BeakerPageObject = require('./beaker.po.js');
 var path = require('path');
 
 describe('autotranslation', function() {
-  beforeEach(function() {
+  beforeEach(function(done) {
     jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000; // Slow initialization cells, CI box might need a long time to initialize these
     beakerPO = new BeakerPageObject();
-    browser.get(beakerPO.baseURL);
-    browser.waitForAngular();
+    browser.get(beakerPO.baseURL).then(
+        function(){done();},
+        function(reason){ console.log('error occurred while browser load page : ' + reason);}
+    );
+    browser.waitForAngular().then(function() {
+      done();
```

Comment:
Anyway Protractor automatically applies browser.waitForAngular() before every WebDriver action.
And sometimes we get exeption "Angular could not be found on the page" when execute  browser.get(beakerPO.baseURL). And after that error Protractor fail all tests.
I give second chance to load Angular.
I can write like this:
browser.get(beakerPO.baseURL)
.then(done, browser.waitForAngular())
.then(done);
Or/And we can set time to wait for Angular to start for browser.get(url, opt_timeout). Default 10 sec. For example we can set 20 sec.
What do you think?